accessibility.html

```diff
@@ -62,7 +62,7 @@ <h2 class="p-heading--three">Accessibility testing</h2>
         <li class="p-list__item is-ticked">The general architecture and hierarchy of the content should make sense</li>
         <li class="p-list__item is-ticked">Charts and images should all have alt-text so that users with screen readers or users on a slow connection will still be able to understand the images</li>
       </ul>
-      <small>*Adapted from <a class="p-link--external" href="http://accessibility.voxmedia.com/">Accessibility Guidelines</a> checklist and <a class="p-link--external" href="http://a11yproject.com/checklist.html">Web Accessibility Checklist</a></small>
```

Comment:
I'm pretty sure trailing slashes on the domain root are irrelevant. removing them probably isn't a problem (shouldn't cause a redirect, 'cos browsers are smart about it), although I think the technically correct form of the URL should have a slash on the end.